integration-tests/src/tests/client/features/delegate_action.rs

```diff
@@ -133,10 +135,12 @@ fn check_meta_tx_execution(
         .get_access_key(&relayer, &PublicKey::from_seed(KeyType::ED25519, relayer.as_ref()))
         .unwrap()
         .nonce;
-    let user_pubk = if sender.is_implicit() {
-        PublicKey::from_implicit_account(&sender).unwrap()
-    } else {
-        PublicKey::from_seed(KeyType::ED25519, sender.as_ref())
+    let user_pubk = match sender.get_account_type() {
+        AccountType::NearImplicitAccount => PublicKey::from_near_implicit_account(&sender).unwrap(),
+        AccountType::EthImplicitAccount => {
+            PublicKey::from_seed(KeyType::SECP256K1, sender.as_ref())
+        }
```

Comment:
What does this do? Please add a TODO if you need to change it later.

Reply:
@wacban  It is to return a public key based on account address. Now I think it is confusing here, as well as `eth_implicit_test_account` in `test_utils`. I am removing it, it is currently unused in integration tests and it will make more sense in the next PR.